src/main/java/de/craftlancer/clstuff/citizensets/CitizenSetsListener.java

```diff
@@ -0,0 +1,222 @@
+    }
+    
+    public void load() {
+        YamlConfiguration config = YamlConfiguration.loadConfiguration(file);
+        
+        citizenSets = (List<CitizenSet>) config.getList("citizenSets", new ArrayList<>());
+    }
+    
+    public void save() {
+        YamlConfiguration config = YamlConfiguration.loadConfiguration(file);
+        
+        config.set("citizenSets", citizenSets);
+        
+        try {
+            config.save(file);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    public CLStuff getPlugin() {
+        return plugin;
+    }
+    
+    @EventHandler()
+    public void onInventoryClose(InventoryCloseEvent event) {
+        Player player = (Player) event.getPlayer();
+        verify(player, 0);
+    }
+    
+    @EventHandler(ignoreCancelled = true)
+    public void onArmorEquip(PlayerInteractEvent event) {
+        Player player = event.getPlayer();
+        PlayerInventory inventory = player.getInventory();
+        List<String> armor = Arrays.asList("HELMET", "CHESTPLATE", "LEGGINGS", "BOOTS");
+        
+        if (event.getAction() != Action.RIGHT_CLICK_BLOCK || event.getAction() != Action.RIGHT_CLICK_AIR)
+            return;
+        
+        if (armor.stream().noneMatch(s -> s.contains(inventory.getItemInMainHand().getType().name()))
+                && armor.stream().noneMatch(s -> s.contains(inventory.getItemInOffHand().getType().name())))
+            return;
+        
+        verify(player, 1);
+    }
+    
+    @EventHandler()
+    public void onHandSwap(PlayerItemHeldEvent event) {
+        Player player = event.getPlayer();
+        verify(player, 1);
+    }
+    
+    @EventHandler
+    public void onItemPickup(EntityPickupItemEvent event) {
+        if (!(event.getEntity() instanceof Player))
+            return;
+        Player player = (Player) event.getEntity();
+        verify(player, 1);
+    }
+    
+    @EventHandler
+    public void onItemDrop(PlayerDropItemEvent event) {
+        Player player = event.getPlayer();
+        verify(player, 0);
+    }
+    
+    @EventHandler
+    public void onItemPickup(PlayerSwapHandItemsEvent event) {
+        Player player = event.getPlayer();
+        verify(player, 1);
+    }
+    
+    @EventHandler()
+    public void onPlayerLeave(PlayerQuitEvent event) {
+        Player player = event.getPlayer();
+        citizenSets.forEach(set -> {
+            if (set.contains(player))
+                set.remove(player.getUniqueId());
+        });
+    }
+    
+    @EventHandler(ignoreCancelled = true)
+    public void onPlayerJoin(PlayerJoinEvent event) {
+        Player player = event.getPlayer();
+        verify(player, 0);
+    }
+    
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGH)
+    public void onPlayerLeave(PlayerRespawnEvent event) {
+        Player player = event.getPlayer();
+        verify(player, 0);
+    }
+    
+    private void verify(Player player, int runLater) {
```

Comment:
Since it's only 1 or 0 apparently exchange it for a boolean, maybe. Makes it more descriptive.